e2e/regression/test-data/mongodb.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mongodb
+  namespace: REPLACE_NAMESPACE
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: mongodb
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: mongodb
+    spec:
+      containers:
+        - name: mongodb
+          securityContext:
+            runAsUser: 101
+          image: quay.io/mongodb/mongodb-community-server@sha256:8b73733842da21b6bbb6df4d7b2449229bb3135d2ec8c6880314d88205772a11
+          volumeMounts:
+            - mountPath: /data/db
+              name: db
+            - mountPath: /data/configdb
+              name: configdb
```

Comment:
We should document all the workaround we had to make the the yaml to get things running. Or should we even revert them and let the test fail until these things are fixed? :thinking:

Reply:
I would say we should leave the fixes in since they should not break things when the bug is fixed eventually. If we document this, the best place would probably be our troubleshooting section in the documentation, I would handle that in a seperate PR if it's fine for you?

Reply:
I would say we should leave the fixes in, since they should not break things when the bug is fixed eventually. If we document this, the best place would probably be our troubleshooting section in the documentation, I would handle that in a separate PR if it's fine for you?